pages/patient-home.tsx

```diff
@@ -1,9 +1,94 @@
+import { useQuery } from '@tanstack/react-query'
+import { createColumnHelper } from '@tanstack/react-table'
+import { useState } from 'react'
+import { getMySubmissions } from '../api/submission'
 import { MainLayout } from '../components/layouts/mainLayout'
+import { BaseTable } from '../components/shared/BaseTable'
+import { Select } from '../components/shared/Select'
+import { PatientSubmission } from '../types/submissions'
 
 export default function PatientHomePage() {
+  const [selectedStatus, setSelectedStatus] = useState('all')
```

Comment:
there's some work we can do here to improve the ts types of this useState, ping me if you're interested